src/utils/todo/task-filing/content-loader.ts

```diff
@@ -102,6 +102,8 @@ export const getDraftTitle = (filing: TaskToDoI): string => {
       return `${t('text.todoItem.addressChange.draftTitle')}`
     case FilingTypes.CHANGE_OF_DIRECTORS:
       return `${t('text.todoItem.directorChange.draftTitle')}`
+    case FilingTypes.CHANGE_OF_OFFICER:
+      return FilingNames.CHANGE_OF_OFFICER
```

Comment:
These are not the enums that you just added, ie:

<img width="292" height="37" alt="image" src="https://github.com/user-attachments/assets/65027b4d-16f0-44aa-a08f-93106cc20ad7" />
<img width="300" height="31" alt="image" src="https://github.com/user-attachments/assets/7d40d75e-4776-4337-bfea-9b64a6f8ee37" />